packages/atomic/.storybook/main.mts

```diff
@@ -47,9 +47,12 @@ const config: StorybookConfig = {
     options: {},
   },
 
-  viteFinal: async (config) =>
+  viteFinal: async (config, {configType}) =>
     mergeConfig(config, {
-      plugins: [nxViteTsPaths(), externalizeDependencies()],
+      plugins: [
+        nxViteTsPaths(),
+        configType === 'PRODUCTION' && isCDN && externalizeDependencies(),
+      ],
```

Comment:
`configType === 'PRODUCTION'` means we run `storybook build`.
We are building the storybook outside the CDN when we run the tests to go fast and leverage build caches.
In this specific case, we do want rollup to be smart and resolve our dependencies (and yes bundle them, but we don't care: it ain't exported)